### PR TITLE
Palak Assignment 14_40Min

### PR DESCRIPTION
**Pull request by PalakSJun20**
1. Frequency OF character from String array working fine
2. can u help me with another method "findCharacterFromArrayAnotherMethod()"
Frequency OF character from String array with Ignore Case
not getting proper out count is not increasing.  

image

### CODE REVIEW

**Review by KrishnaKTechnocredits — APPROVED** [review]
Will send you voice msg.

File: src/palakSJun20/FrequencyOfCharacter.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	///Finding Character from Array String using Ignore Case 
+	void findCharacterFromArrayAnotherMethod(String[] array , char findCh){
+		String name = null;
+		int count = 0;
+		String findChar = String.valueOf(findCh);
+		for (int i = 0; i < array.length; i++) {
+			name = array[i];
+			
+			for (int j = 0; j < name.length(); j++) {
+				if(name.equalsIgnoreCase(findChar))
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/FrequencyOfCharacter.java]: String.valueOf(name.charAt(j)).equalsIgnoreCase(findChar)

Comment (PalakSJun20) [reply src/palakSJun20/FrequencyOfCharacter.java]: Understood Changes Updated
Thanks

Comment (KrishnaKTechnocredits) [reply src/palakSJun20/FrequencyOfCharacter.java]: Okay, great.

**Pull request merged**